CuratedContent/WorkingEffectivelyWithLegacyCode.md

```diff
@@ -57,15 +58,15 @@ Targeted changes should be done in small iterations so this should hopefully jus
 You may need to add "sensing" variables to help see what you need to see in a unit test.
 
 * **Break dependencies**: Dependencies need to be broken for one of two reasons: Sensing and Separation.  With **Sensing**, one must be able to inspect the behavior of the code that we can’t otherwise see.  While **Separation** is needed to allow the code to be run in a test harness outside of the production setting.
-Actually breaking the dependencies involves doing minimal refactorings with careful hyper-sensitive editing.
+Actually breaking the dependencies involves doing minimal refactoring with careful hyper-sensitive editing.
```

Comment:
It is okay to say either "refactoring" or "refactorings" given the meanings of these words in the SE community.  I kink of like referring to a collection "refactorings" rather than just that activity of "refactoring".